project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.15
+sbt.version=0.13.16-M1
```

Comment:
We released 0.13.16-M1 because I wanted `^^` facility to be available to plugin authors without going through the RC cycle. I haven't pushed for final because I wanted to make sure that sbt 0.13.16 final is able to cross build against sbt 1.0.0. Now that sbt 1.0.0-RC2 is out we probably can start the process.

The only reported compatibility issue is the forward compatibility breakage of the ScriptedPlugin in https://github.com/foundweekends/giter8/issues/325. This I think is a corner case because Giter8's plugin depends on Scripted directly - https://github.com/foundweekends/giter8/blob/547db187ceadb4cefc504733ce47225f7b7b47aa/build.sbt#L97